platform/accessibility/README.md

```diff
@@ -1,24 +1,77 @@
+*Last updated: March 20, 2020*
+*Owned by: VSP Accessibility*
+
 # Accessibility
-In this folder, you'll find information specific to how 508 Accessibility works on VA.gov.
 
-* Learn about [accessibility best practices](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-accessibility-best-practices.md) in design and engineering
-* Learn how to request a [508 pre-launch review](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/508-request-prelaunch-review.md) for your product or feature improvement
+VSP Accessibility supports teams building on VA.gov to ensure that nobody is excluded from using products on VA.gov. We provide best practice guidance throughout the development cycle, build and recommend automated testing tools to flag accessibility issues, reducing the manual effort required to fix issues immediately before launch and post-la
+
```

Comment:
Looks like content is missing here.

Reply:
It has been moved further down the page, in our Resources and Guides section.